src/ucp/rndv/rndv.inl

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (C) 2022, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCP_RNDV_RNDV_INL_
+#define UCP_RNDV_RNDV_INL_
```

Comment:
rndv.h already has some inline function, so either move them here, or move ucp_rndv_receive_start to rndv.h

Reply:
Moved inline functions to rndv.inl, because it is required to include more files to .h file, otherwise.